src/MessageHandler.js

```diff
@@ -1,294 +1,303 @@
-import { Component } from 'react';
-import YAML from 'yaml'
-import rawJackboxGameList from './JackboxGames.yaml';
-const fetch = require('node-fetch');
-const tmi = require('tmi.js');
-
-const GAME_REQUEST_COMMAND = "!request";
-
-export default class MessageHandler extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      client: null,
-      validGames: []
-    };
-  }
-
-  componentDidMount(props) {
-    const client = new tmi.client({
-      identity: {
-        username: this.props.channel,
-        password: this.props.access_token
-      },
-      channels: [
-        this.props.channel
-      ]
-    });
-
-    client.on('message', this.onMessage);
-    client.connect();
-
-    fetch(rawJackboxGameList)
-      .then(r => r.text())
-      .then(text => {
-        this.setState((state) => {
-          return {
-            ...state,
-            client,
-            validGames: YAML.parse(text)
-          };
-        });
-      })
-    }
-
-  isModOrBroadcaster = (username) => {
-    return (this.props.channel === username || this.props.modList.includes(username.toLowerCase()));
-  }
-
-  // returns true iff a known command was found & responded to

```

Comment:
lol, that wasn't a typo -- iff=["if and only if"](https://en.wikipedia.org/wiki/If_and_only_if) -- but no big deal!

Reply:
Ohhhhh! I figured it was a typo XD